### PR TITLE
[release-v0.26.0] Add kn-event-image to spec file

### PR DESCRIPTION
**Pull request by dsimansk**
/cc @Kaustubh-pande @cardil

### CODE REVIEW

File: openshift-serverless-clients.spec [hunk 1]
```diff
@@ -6,6 +6,7 @@
 %global kn_version 0.26.0
 %global kn_release 1
 %global kn_cli_version v%{kn_version}
+%global kn_event_image sender-nvr
```
Comment (cardil) [inline openshift-serverless-clients.spec:9]:
```suggestion
%global kn_event_image %{nil}
```

https://stackoverflow.com/a/3919298/844449

File: openshift-serverless-clients.spec [hunk 2]
```diff
@@ -28,7 +29,7 @@ Client kn provides developer experience to work with Knative Serving APIs.
 %setup -q -n %{source_dir}
 
 %build
-TAG=%{kn_cli_version} make build-cross
+TAG=%{kn_cli_version} KN_PLUGIN_EVENT_SENDER_IMAGE=%{kn_event_image} make build-cross
```
Comment (cardil) [inline openshift-serverless-clients.spec:32]:
```suggestion
TAG=%{kn_cli_version} \
KN_PLUGIN_EVENT_SENDER_IMAGE=%{kn_event_image} \
make build-cross
```

For better readability

**Review by cardil — CHANGES_REQUESTED** [review]
Some nits

File: openshift-serverless-clients.spec (hunk 1 above)
Comment (dsimansk) [reply openshift-serverless-clients.spec:9]: @Kaustubh-pande what does the tooling say? Would it be able to work with `%{nil}`?

**Review by cardil — APPROVED** [review]
/lgtm

**Pull request merged**